verilog/formatting/tree_unwrapper.cc

```diff
@@ -1361,6 +1362,14 @@ void TreeUnwrapper::ReshapeTokenPartitions(
         break;
       }
 
+    case NodeEnum::kPreprocessorDefine: {
+        auto& last = *ABSL_DIE_IF_NULL(partition.RightmostDescendant());
+        if (PartitionIsCloseParen(last)) {
+          verible::MergeLeafIntoPreviousLeaf(&last);
+        }
+	break;
+      }
```

Comment:
Is this logic needed?  According to verilog.y, a `kPreprocessDefine` node always ends with a `verilog_tokentype::PP_define_body` (which could be empty text).  

Just a note, functions and tasks introduce an auxiliary `Header` node to group together everything up to the formal ports.  If you feel it is needed, you could separate out a Header nodes for macro definitions, but the only thing it would exclude is the `PP_define_body`.

Reply:
Without this condition the following tests fails:
```
`define FOO
`define BAR
```
like this:
```
`define FOO
    `define BAR
```

Reply:
Added this note